tests/integration_tests/build/test_binary_size.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests that check if the release binary sizes fall within expected size"""
+
+import os
+import pytest
+import platform
+
+import host_tools.cargo_build as host
+
+MACHINE = platform.machine()
+""" Platform definition used to select the correct size target"""
+
+FC_BINARY_SIZE_TARGET = 3137280 if MACHINE == "x86_64" else 3536544
+"""Firecracker target binary size to be checked in bytes"""
+
+JAILER_BINARY_SIZE_TARGET = 2985152 if MACHINE == "x86_64" else 3149128
+"""Jailer target binary size to be checked in bytes"""
+
+BINARY_SIZE_TOLERANCE = 0.05
+"""Tolerance of 5% allowed for binary size"""
+
+FEATURES = [""]
```

Comment:
We don't have any features, might as well remove this and the parametrization to keep the code as simple as possible. We can always re-add it if we feature-ize again in the future

Reply:
Done